### PR TITLE
TASK-28: Create README.md front-end

### PR DESCRIPTION
**Pull request by acjoell**
# Description:

## Introduction
Create the frontend readme.md file with the following information:
   - Getting Started (how to start with the project)
   - Technology Stack (The components with which it was created)
   - Contribution (Link to the contributing file)
   - Team (name and function of team people)

#

## Conclusion

- Described the prerequisites to start the repo
- Described how you should do the installation of the dependencies
- I described the executions that I can do within the project (both back-end and front-end to date)

- I described how we had formed our technological stack, both in front, back, design and marketing (but it is empty)

- I put two contribution files (CONTRIBUTING.md and CODE-OF CONDUCT) for you to review and rate, they are based on the Mozilla Code of Conduct Application Scale, it seemed like a good idea to me

- Described the team that started the project

- The license space is still empty, due to my ignorance of it

- And a section of Acknowledgments that is empty

### CODE REVIEW

File: CODE-OF-CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+en espacios públicos. Ejemplos de representación de un proyecto o comunidad
+incluir el uso de una dirección de correo electrónico oficial del proyecto, la publicación a través de un
+cuenta oficial de redes sociales, o actuando como representante designado
+en un evento en línea o fuera de línea. La representación de un proyecto puede ser
+más definido y aclarado por los encargados del mantenimiento del proyecto.
+
+## Ejecución
+
+Los casos de comportamiento abusivo, acosador o inaceptable pueden
+ser informado poniéndose en contacto con el equipo del proyecto en <billie@billiecodes.com>.
```
Comment (CesarJOritzM) [inline CODE-OF-CONDUCT.md:61]: the email must be the one from atacha

File: CODE-OF-CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+las políticas pueden publicarse por separado.
+
+Los mantenedores de proyectos que no siguen o hacen cumplir el Código de Conducta en
+La buena fe puede enfrentar repercusiones temporales o permanentes según se determine.
+por otros miembros de la dirección del proyecto.
+
+## Atribución
+
+Este Código de conducta está adaptado del [Colaborador
+Covenant] (https://www.contributor-covenant.org), versión 1.4, disponible
```
Comment (CesarJOritzM) [inline CODE-OF-CONDUCT.md:75]: You must direct to the atacha page

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+## **Atribución**
+
+Este código de conducta está adaptado del [Pacto del colaborador](#) en la [página de inicio](#),
+versión [v2.0](https://www.contributor-covenant.org/version/2/0/code_of_conduct.html), disponible en [contributor covenant](https://www.contributor-covenant.org).
+
+Las pautas de impacto comunitario se inspiraron en la [Escala de aplicación del código de conducta de Mozilla](https://github.com/mozilla/diversity).
+
+Para obtener respuestas a preguntas comunes sobre este código de conducta, consulte las preguntas frecuentes en
+[FAQ](https://www.contributor-covenant.org/faq). Hay traducciones disponibles
+en [translations](https://www.contributor-covenant.org/translations).
```
Comment (CesarJOritzM) [inline CONTRIBUTING.md:131]: You must direct to the Atacha page

**Review by CesarJOtizM — CHANGES_REQUESTED** [review]
I consider that it should only carry the information regarding the front-end, since the detail of the back-end will be done in the README.md of it

File: README.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+
+El stack de tecnologías que usamos es el siguiente:
+
+_Front-end_
+
+* [Next.js - React  ](https://nextjs.org/) - The React Framework for Production
+* [Sass](https://sass-lang.com/) - Stylesheet language that’s compiled to CSS
+* [Apollo Client](https://www.apollographql.com/docs/react/) - Data management library
+
+_Back-end_
```
Comment (CesarJOritzM) [inline README.md:101]: only front-end information

File: README.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+```sh
+npm install #or
+yarn
+```
+
+### **3. Ejecutando proyecto** ⚙️
+
+> _Dentro de la carpeta Atacha-API inicia el modo desarrollador con el siguiente código:_
+
+**1.** Modo desarrollo dentro del Back-end
```
Comment (CesarJOtizM) [inline README.md]: Only front-end information

File: CODE-OF-CONDUCT.md (hunk 1 above)
Comment (CesarJOtizM) [inline CODE-OF-CONDUCT.md]: the email must be the one from atacha

File: CODE-OF-CONDUCT.md (hunk 2 above)
Comment (CesarJOtizM) [inline CODE-OF-CONDUCT.md]: You must direct to the atacha page

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+
+## **Atribución**
+
+Este código de conducta está adaptado del [Pacto del colaborador](#) en la [página de inicio](#),
+versión [v2.0](https://www.contributor-covenant.org/version/2/0/code_of_conduct.html), disponible en [contributor covenant](https://www.contributor-covenant.org).
+
+Las pautas de impacto comunitario se inspiraron en la [Escala de aplicación del código de conducta de Mozilla](https://github.com/mozilla/diversity).
+
+Para obtener respuestas a preguntas comunes sobre este código de conducta, consulte las preguntas frecuentes en
+[FAQ](https://www.contributor-covenant.org/faq). Hay traducciones disponibles
```
Comment (CesarJOtizM) [inline CONTRIBUTING.md]: You must direct to the Atacha page

File: README.md (hunk 4 above)
Comment (CesarJOtizM) [inline README.md]: only front-end information

**Review by CesarJOtizM — CHANGES_REQUESTED** [review]
I consider that it should only carry the information regarding the front-end, since the detail of the back-end will be done in the README.md of it

**Review by CesarJOtizM — CHANGES_REQUESTED** [review]
I consider that it should only carry the information regarding the front-end, since the detail of the back-end will be done in the README.md of it

File: README.md [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+
+<a name="acknowledgments"></a>
+
+#
+## **Acknowledgments** 🎁
+
+* Comenta a otros sobre este proyecto 📢
+* Invita una cerveza 🍺 o un café ☕ a alguien del equipo. 
+* Da las gracias públicamente 🤓.
+* etc.
```
Comment (ruloCode) [inline README.md]: ¿Podrías quitar este etc ?

**Review by ruloCode — CHANGES_REQUESTED** [review]
Los documentos están utilizando inglés y español en desorden. 


image


Si estás considerando agregar el español sería mejor tener la versión original en inglés y una alterna solo en **español.**
image

File: CONTRIBUTING.md (hunk 6 above)
Comment (ruloCode) [reply CONTRIBUTING.md]: La guía de contribución  me debe ser de utilidad en:
- Cómo crear un Pull request
- Cómo nombrar una rama 
- Cómo nombrar un commit
- El proceso de un PR hasta producción

Comment (ruloCode) [reply CONTRIBUTING.md]: este recurso podría ser de ayuda
https://github.com/Sphinx-Society/kaizen-web-client/blob/master/CONTRIBUTING.md

File: README.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+
+> _Inside the "Atacha-website" folder, start the following code:_
+
+**1.** Running in production mode on the Front-end
+```sh
+npm run start #or
+yarn start
+```
+
+**2.** Run production mode on the Front-end
```
Comment (CesarJOtizM) [inline README.md]: Run build in front-end

**Review by CesarJOtizM — CHANGES_REQUESTED** [review]
Change name point 2

**Review by CesarJOtizM — APPROVED** [review]

**Review by ruloCode — APPROVED** [review]
LGTM (Look go to me)

**Pull request merged**